src/TestFileSupport.cpp

```diff
@@ -0,0 +1,59 @@
+#include "TestFileSupport.hpp"
+#include <system_error>
+#include <cerrno>
+
+bool TestFileSupport::CreateTestFiles(std::string inputfiles_directory)
+{
+    //TODO: add Error handling
+    errno = 0;
+    std::error_code ec(errno, std::generic_category());
```

Comment:
Error code is not a good way to handle errors. Generally it is not the way how it is normally used. It is usually used in some system function calls. We have exceptions for this case. Don't forget to remove includes when you get rid of error_code

Reply:
I removed error_code. In this particular case I used return value to determine pass or fail of execution function "remove_all". I think in this particular situation we do not need to use exceptions because failing during create/remove directory is not some special situation. 

Done